OpenChain Telco SBOM Specification.md

```diff
@@ -81,6 +81,7 @@ Package information
 * PackageSupplier: needed by “NTIA SBOM Minimum elements”
 * PackageDownloadLocation: mandatory in SPDX
 * FilesAnalyzed
+* PackageChecksum: recommended by “NTIA SBOM Minimum elements”
```

Comment:
I guess that the order does not matter that much. In the SPDX Annex (https://spdx.github.io/spdx-spec/v2.3/how-to-use/#k21-us-executive-order-14028-minimum-elements-for-an-sbom) is after package version, but as said, I do not think that the order is that important.

Reply:
I used the order of the SPDX standard.